ui/admin/tests/acceptance/workers/list-test.js

```diff
@@ -191,26 +198,34 @@ module('Acceptance | workers | list', function (hooks) {
     });
 
     await visit(urls.workers);
-    await click(WORKER_TAGS_BUTTON(instances.worker.id));
 
-    assert.dom(WORKERS_FLYOUT_TABLE_ROWS).exists({ count: 5 });
-    assert.dom(WORKERS_FLYOUT_VIEW_MORE_TAGS).isNotVisible();
+    await click(
+      selectors.TABLE_ROW_WORKER_TAGS_FLYOUT_BUTTON(instances.worker.id),
+    );
+
+    assert.dom(selectors.WORKER_TAGS_FLYOUT_TABLE_ROWS).exists({ count: 5 });
+    assert
+      .dom(selectors.WORKER_TAGS_FLYOUT_VIEW_MORE_TAGS_BUTTON)
+      .isNotVisible();
```

Comment:
nitpick: Instead of using `isNotVisible()` could we use `doesNotExist()`. This is more around the intent with the test. In this instance, the "View more tags" button shouldn't exist at all if the right conditions are met. The `isNotVisible()` assert reads more as, it may exist but we have temporarily hidden it. I know this was already being used which is why I say nitpick. I will add this as a tech time topic for next week.